README.md

```diff
@@ -367,45 +367,46 @@ K8s secrets can also be passed to the function using the `secureParameters` fiel
 Here, the `secret` value is the K8s secret name that will be mounted into the runtime and made available to the function
 via the environment variable `SECURE_DATA`.
 
-### Keptn Task
+### KeptnTask
 
 A Task is responsible for executing the TaskDefinition of a workload.
 The execution is done spawning a K8s Job to handle a single Task.
 In its state, it keeps track of the current status of the K8s Job created.
 
-### Keptn Evaluation Definition
+### KeptnEvaluationDefinition
 
 A `KeptnEvaluationDefinition` is a CRD used to define evaluation tasks that can be run by the Keptn Lifecycle Toolkit
 as part of pre- and post-analysis phases of a workload or application.
 
-A Keptn evaluation definition looks like the following:
+A KeptnEvaluationDefinition looks like the following:
 
 ```yaml
 apiVersion: lifecycle.keptn.sh/v1alpha2
 kind: KeptnEvaluationDefinition
 metadata:
   name: my-prometheus-evaluation
 spec:
-  source: prometheus
   objectives:
-    - name: query-1
-      query: "xxxx"
+    - keptnMetricRef:
```

Comment:
are those changes already in `lifecycle.keptn.sh/v1alpha2`? I think we need to bump to alpha3 here no?

Reply:
you are right, will fix, thanks!

Reply:
fixed